Docs/source/pages/advanced-usage.md

```diff
@@ -314,6 +347,34 @@ In the routing table, the personal hotspot network would be secondary, therefore
 
                 Not available. A workaround is described above.
 
+    .. tab:: robot
+
+        .. tabs::
+
+            .. tab:: Android
+
+                Use the following static methods from the **AltReversePortForwarding** class in your test file:
+
+                    - **Reverse Port Forwarding Android**
+                        .. code-block:: robot
+
+                            Reverse Port Forwarding Android    device_port=13000    local_port=13000    device_id= 
+                    
```

Comment:
You could use `device_id=your_device_id`

Reply:
fixed